connectors/connectors.sh

```diff
@@ -85,6 +100,16 @@ fi
 update_connector "bigquery" "$BIGQUERY_CONNECTOR_VERSION"
 update_connector "gcs" "$GCS_CONNECTOR_VERSION"
 
+if [[ $UPDATED_GCS_CONNECTOR != true ]]; then
+  echo "GCS connector wasn't updated - no need to restart any services"
+  exit 0
```

Comment:
you might just want this to be a return in case the caller expects something after this function

Reply:
This is not a function,